src/main/java/hudson/plugins/gradle/injection/MavenInjectionAware.java

```diff
@@ -0,0 +1,96 @@
+package hudson.plugins.gradle.injection;
+
+import hudson.EnvVars;
+import hudson.model.InvisibleAction;
+import hudson.model.Node;
+import hudson.model.Run;
+import hudson.model.TaskListener;
+import hudson.plugins.gradle.util.CollectionUtil;
+
+import java.io.IOException;
+import java.util.Objects;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public interface MavenInjectionAware {
+
+    String JENKINSGRADLEPLUGIN_MAVEN_OPTS_PREPARED = "JENKINSGRADLEPLUGIN_MAVEN_OPTS_PREPARED";
+    String JENKINSGRADLEPLUGIN_MAVEN_PLUGIN_CONFIG_EXT_CLASSPATH_PREPARED = "JENKINSGRADLEPLUGIN_MAVEN_PLUGIN_CONFIG_EXT_CLASSPATH_PREPARED";
```

Comment:
Same as with `JENKINSGRADLEPLUGIN_MAVEN_OPTS`. I don't think that `_PREPARED` does bring any value here.

Reply:
See my comment above